app/views/councilTaxUpload.scala.html

```diff
@@ -29,6 +29,13 @@
 
 @(baCode: String, fileUploadForm: Form[_], initiateResponse: Option[InitiateResponse] = None)(implicit request: Request[_], messages: Messages)
 
+@label = {
+
+<h1 class="govuk-label-wrapper" id="heading">
```

Comment:
Needed to add an id to the Label (to inject some html after it when needed), couldn't seem to do it with the api from `play-frontend-govuk`